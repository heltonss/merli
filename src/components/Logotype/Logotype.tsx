import Logo from 'assets/images/Logo_ML.png';
import Logo2x from 'assets/images/Logo_ML@2x.png';
import React from 'react';

interface IOwnProps {
  parentClassStyle?: string;
}

const Logotype = ({ parentClassStyle }: IOwnProps) => {
  return (
    <img
      className={parentClassStyle}
      src={Logo}
      alt="logotipo mercado livre"
      srcSet={`${Logo2x} 2x`}
    />
  );
};

export default Logotype;
