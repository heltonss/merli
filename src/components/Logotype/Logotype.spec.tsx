import { shallow } from 'enzyme';
import React from 'react';
import Logotype from './Logotype';

describe('Logotype ', () => {
  it('Should render without crash', () => {
    const wrapper = shallow(<Logotype />);

    expect(wrapper).toMatchSnapshot();
  });
});
