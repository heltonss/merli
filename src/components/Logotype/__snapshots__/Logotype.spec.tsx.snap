// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logotype  Should render without crash 1`] = `ShallowWrapper {}`;
